conda_build/main_build.py

```diff
@@ -185,10 +185,28 @@ def main():
         metavar="R_VER",
         choices=RVersionsCompleter(),
     )
+    p.add_argument(
+        '-v',
+        '--verbose',
+        action='store_true',
+        help="Enable verbose mode. Turns on extra debug output"
+    )
     add_parser_channels(p)
     p.set_defaults(func=execute)
 
     args = p.parse_args()
+    # enable logging. Default to warning
+    import logging
+    stream = logging.StreamHandler()
```

Comment:
If memory serves, `StreamHandler` defaults to `stderr`.  If that's not what you intended, you should write `StreamHandler(sys.stdout)`.